src/main/java/science/icebreaker/account/Profile.java

```diff
@@ -0,0 +1,99 @@
+package science.icebreaker.account;
+
+
+import org.springframework.lang.Nullable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+@Entity
+public class Profile {
```

Comment:
To be consistent, we should rename this class AccountProfile analog to AccountRole